scripts/functions/def_offset.py

```diff
@@ -0,0 +1,159 @@
+#------------------------------------------------------------------------------------------------------------------------------
+# By Alexandra Lee 
+# (updated October 2018) 
+#
+# Define offset vectors
+#-------------------------------------------------------------------------------------------------------------------------------
+import os
+import pandas as pd
+import numpy as np
+from keras.models import model_from_json, load_model
+from keras import metrics, optimizers
+
+randomState = 123
+from numpy.random import seed
+seed(randomState)
+
+def gene_space_offset(data_dir, gene_id):
+    """
+    Gene space offset:
+
+    offset_vector = average(gene expression of samples that have the highest 5% of target gene expression) -  
```

Comment:
This information is now added.  The target gene is used to sort the samples and can be thought of as your "phenotype".  In "interpolate_<>.py", after we sort samples based on the expression level of the target gene, we want to predict the expression profile of the OTHER genes at different levels of target gene expression.  Hopefully that is a bit more clear.  It is a bit confusing because we are using expression to predict expression.